httpclient/src/main/java/org/apache/http/client/utils/URIBuilder.java

```diff
@@ -492,8 +492,8 @@ public String toString() {
 
     private static String normalizePath(final String path) {
         String s = path;
-        if (s == null) {
-            return "/";
+        if (s == null || "".equals(s)) {
+            return "";
```

Comment:
Hi garygregory,
s == null || "".equals(s) has been changed to TextUtils.isBlank(s), which would be better. isn't it?